spi/contract-spi/src/main/java/org/eclipse/dataspaceconnector/spi/contract/policy/store/PolicyStore.java

```diff
@@ -0,0 +1,64 @@
+/*
+ *  Copyright (c) 2020 - 2022 Microsoft Corporation
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Microsoft Corporation - initial API and implementation
+ *
+ */
+package org.eclipse.dataspaceconnector.spi.contract.policy.store;
```

Comment:
Since the `Policy` class is in the `core-spi`, maybe also this store will fit better there? Maybe could be time for a `policy-spi` module?

Reply:
I created policy-spi in spi and moved the PolicyStore there to be consistent with other store interfaces